src/WebJobs.Script.WebHost/Resources/Functions/WarmUp/run.csx

```diff
@@ -1,9 +1,26 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿#r "Newtonsoft.Json"
+#r "Microsoft.WindowsAzure.Storage"
+
+
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Newtonsoft.Json;
 
 public static IActionResult Run(HttpRequest req, TraceWriter log)
 {
     log.Info("WarmUp function invoked.");
 
-    return (ActionResult)new OkObjectResult("WarmUp complete.");
+    string name = req.Query["name"];
 
-}
\ No newline at end of file
+    string requestBody = new StreamReader(req.Body).ReadToEnd();
+    dynamic data = JsonConvert.DeserializeObject(requestBody);
+    name = name ?? data?.name ?? "";
+
+    // This is just to warmup storage calls in placeholder process to improve cold startup time.
+    CloudBlockBlob output = new CloudBlockBlob(new Uri("https://blahblah.blob.core.windows.net/blahblah/"));
```

Comment:
This will fail auth right, since you're not using a client with account credentials? Does it actually perform a storage operation attempt or does it fail early client side?

Reply:
Can we use an invalid URI here? We could also use a valid URI for an account we own (e.g., we expose feeds and things like that from a public storage account. We could have a blob we hit there)

Reply:
It fails early on and we don’t care. But if the URI is invalid it doesn’t go thru the client initializaion which we need.

Reply:
We should consider doing what I mentioned, though. We should use the URL for an account we own, as this is a completely valid name and could be hitting a valid storage account we don’t